cowsay/main.py

```diff
@@ -1,93 +1,205 @@
-from __future__ import print_function
-import sys
-import re
+from __future__ import print_function # for backwards compatibility
 
+import sys
 from .characters import CHARS
 
 __version__ = '5.0'
 
 char_names = list(CHARS.keys())
 
+def wrap_lines(
+    lines: list,
+    max_width: int = 49
+    ) -> list:
+    """
+    wrap_lines(lines, max width) -> wrapped lines
+
+    Wraps all lines in a box, so it doesn't overflow
+
+    :param lines list: List of lines
+    :param max_width int: Max width
+    :returns list: Wrapped lines
+    """
 
-def wrap_lines(lines, max_width=49):
     new_lines = []
+
     for line in lines:
         for line_part in [
             line[i:i+max_width] for i in range(0, len(line), max_width)
-        ]:
+            ]:
             new_lines.append(line_part)
+
     return new_lines
 
+def generate_bubble(
+    text: str
+    ) -> list:
+    """
+    generate_bubble(text) -> bubble
+
+    Creates a chat bubble around the text
+
+    :param text str: Text to put in the bubble
+    :returns list: The bubble
+    """
+
+    lines = [
+        line.strip()
+        for line in str(text).split("\n")
+    ]
+
+    lines = wrap_lines([
+        line
+        for line in lines
+        if line
```

Comment:
Instead of having `if line` here why not add it in the previous list comp?

Reply:
> Instead of having `if line` here why not add it in the previous list comp?

Just fixed that together with some minor inconsistencies, thanks!